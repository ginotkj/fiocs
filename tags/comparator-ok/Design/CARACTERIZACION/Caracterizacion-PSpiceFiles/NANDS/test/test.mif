 lib=D:\Documents\TESIS\fiocs\Design\CARACTERIZACION\caracterizacion-pspicefiles\caracterizacion.lib, offset=42, size=2422
 lib=D:\Documents\TESIS\fiocs\Design\CARACTERIZACION\caracterizacion-pspicefiles\caracterizacion.lib, offset=2465, size=2440
