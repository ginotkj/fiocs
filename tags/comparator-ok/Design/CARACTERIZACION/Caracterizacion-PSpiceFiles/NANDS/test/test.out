
**** 04/17/10 18:28:15 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "NANDS-test"  [ D:\Documents\TESIS\fiocs\Design\CARACTERIZACION\caracterizacion-pspicefiles\nands\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../caracterizacion-pspicefiles/caracterizacion.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 250n 0 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\NANDS.net" 



**** INCLUDING NANDS.net ****
* source CARACTERIZACION
R_R4         NAND4 N00173  1Meg  
R_R1         NEG N00173  1Meg  
V_V1         ENTRADA 0  
+PULSE 0 3.3 0 100n 100n 50n 250n
R_R7         NAND8 N00173  1Meg  
M_U1_M16         NAND8 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M5         NAND8 ENTRADA N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M11         NAND8 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M9         NAND8 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M1         NAND8 N00173 U1_ND2 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U1_M6         U1_ND3 N00173 U1_ND4 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U1_M7         U1_ND4 N00173 U1_ND5 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U1_M12         NAND8 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M2         U1_ND2 N00173 U1_ND3 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U1_M13         U1_ND6 N00173 U1_ND7 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U1_M10         U1_ND5 N00173 U1_ND6 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U1_M3         NAND8 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M14         U1_ND7 N00173 U1_ND8 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U1_M8         NAND8 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M4         NAND8 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U1_M15         U1_ND8 ENTRADA 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U5_M7         NAND4 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U5_M8         NAND4 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U5_M12         NAND4 ENTRADA N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U5_M9         NAND4 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U5_M1         NAND4 N00173 U5_ND2 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U5_M10         U5_ND3 N00173 U5_ND4 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U5_M3         U5_ND2 N00173 U5_ND3 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U5_M11         U5_ND4 ENTRADA 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
R_R2         NAND2 N00173  1Meg  
M_U2_M5         NAND2 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U2_M6         NAND2 N00173 U2_ND2 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U2_M2         U2_ND2 ENTRADA 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U2_M4         NAND2 ENTRADA N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
R_R6         NAND7 N00173  1Meg  
M_U3_M5         NEG ENTRADA N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U3_M6         NEG ENTRADA 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U7_M1         NAND7 N00173 U7_ND2 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U7_M14         U7_ND7 ENTRADA 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U7_M8         NAND7 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U7_M2         U7_ND2 N00173 U7_ND3 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U7_M3         NAND7 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U7_M4         NAND7 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U7_M11         NAND7 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U7_M7         U7_ND4 N00173 U7_ND5 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U7_M9         NAND7 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U7_M6         U7_ND3 N00173 U7_ND4 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U7_M12         NAND7 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U7_M5         NAND7 ENTRADA N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U7_M10         U7_ND5 N00173 U7_ND6 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U7_M13         U7_ND6 N00173 U7_ND7 0 CMOSN  
+ L=.18u  
+ W=.18u         
V_VDD         N00173 0 3.3
M_U6_M4         NAND5 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U6_M9         NAND5 ENTRADA N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U6_M7         U6_ND4 N00173 U6_ND5 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U6_M10         U6_ND5 ENTRADA 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U6_M5         NAND5 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U6_M8         NAND5 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U6_M1         NAND5 N00173 U6_ND2 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U6_M6         U6_ND3 N00173 U6_ND4 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U6_M2         U6_ND2 N00173 U6_ND3 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U6_M3         NAND5 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
R_R3         NAND3 N00173  1Meg  
M_U4_M2         U4_ND2 N00173 U4_ND3 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U4_M3         NAND3 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U4_M6         U4_ND3 ENTRADA 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U4_M4         NAND3 N00173 N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
M_U4_M1         NAND3 N00173 U4_ND2 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_U4_M5         NAND3 ENTRADA N00173 N00173 CMOSP  
+ L=.18u  
+ W=.2u         
R_R5         NAND5 N00173  1Meg  

**** RESUMING test.cir ****
.END

Mosfet M_U1_M16, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U1_M16, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U1_M16, model CMOSP: Ps = 0 is less than W
Mosfet M_U1_M1, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U1_M1, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U1_M1, model CMOSN: Ps = 0 is less than W
**** 04/17/10 18:28:15 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "NANDS-test"  [ D:\Documents\TESIS\fiocs\Design\CARACTERIZACION\caracterizacion-pspicefiles\nands\test.sim ] 


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               CMOSP           CMOSN           
               PMOS            NMOS            
  T_Measured   27              27            
   T_Current   27              27            
       LEVEL    7               7            
           L  100.000000E-06  100.000000E-06 
           W  100.000000E-06  100.000000E-06 
         VTO    -.413091         .294194     
          KP  460.419200E-06  460.419200E-06 
       GAMMA    0               0            
      LAMBDA    0               0            
         RSH    5.9             6.2          
          IS    1.000000E-15    1.000000E-15 
          JS  100.000000E-06  100.000000E-06 
          PB     .821464         .99         
        PBSW     .8008           .8          
          CJ    1.165977E-03  815.244000E-06 
        CJSW  122.005600E-12  165.796200E-12 
          MJ     .425655         .610228     
        MJSW     .1001           .163262     
        CGSO  500.000000E-12  500.000000E-12 
        CGDO  500.000000E-12  500.000000E-12 
        CGBO    1.000000E-12    1.000000E-12 
         TOX    4.500000E-09    4.500000E-09 
          XJ  100.000000E-09  100.000000E-09 
       UCRIT   10.000000E+03   10.000000E+03 
       DELTA     .01             .01         
      DIOMOD    2               2            
          K1     .59909          .541355     
          K2 -669.794000E-09    -.032332     
        LETA    0               0            
        WETA    0               0            
          U0  118.3014        269.8792       
       XPART     .5              .5          
        VTH0    -.413091         .294194     
          K3     .080859        1.000406E-03 
          W0    1.000000E-06  100.000000E-09 
         NLX   25.530790E-09  206.491900E-09 
        DVT0     .819884         .565214     
        DVT1     .744212         .260305     
          UA    1.535614E-09   -1.619209E-09 
          UB    1.000000E-21    2.953688E-18 
          UC -100.000000E-12   57.637850E-12 
        VSAT  173.454300E+03  106.504000E+03 
        RDSW  677.8533        150            
        VOFF    -.142681        -.09299      
     NFACTOR     .94145         2.044734     
        PCLM     .334199        1.968647     
      PDIBL1     .132793        1            
      PDIBL2    9.825629E-03    -.072042     
       DROUT     .81589         1            
      PSCBE1    3.057881E+09    2.350722E+09 
      PSCBE2    5.892816E-09  508.313600E-12 
          A0     .926787        1.425322     
          A1     .01033          .023731     
          A2     .752154         .550469     
       NPEAK  415.890000E+15  235.490000E+15 
         LDD    0               0            
        LITL   36.742350E-09   36.742350E-09 
         UC1  -56.000000E-12  -56.000000E-12 
        PVAG     .122566                     
        KETA     .022875        -.016685     
        ETA0    1.263659E-03    6.051176E-03 
        ETAB   -2.195822E-03 -858.512600E-06 
         K3B   19.71403         5.01511      
        DVT2    -.3             -.067374     
        DSUB    4.891602E-03    3.642935E-03 
         AGS     .166834         .37123      
       DVT1W    0               0            
       DVT2W    0               0            
        PRWG    -.054897         .34061      
        PRWB    -.5             -.183689     
     PDIBLCB   -1.000000E-03    -.1          
         DWG  -25.031010E-09    5.313945E-09 
         DWB  -12.379610E-09   13.440770E-09 
          B0    1.874426E-06  -23.699750E-09 
          B1    5.000000E-06                 
        LINT   27.788770E-09   20.673220E-09 
        WINT                   12.447730E-09 
         DLC   27.788770E-09   20.673220E-09 
         DWC    0              12.447730E-09 
          CF    0               0            
        NOIA    9.900001E+18  100.000000E+18 
        NOIB    2.400000E+03   50.000000E+03 
        NOIC    1.400000E-12   -1.400000E-12 
       LKETA    -.010867      399.601500E-06 
       WKETA     .03239         3.401938E-03 
       PVSAT   50             790.3922       
       PKETA   -6.167985E-03    -.014873     
       PVTH0    1.001339E-03   -4.814415E-03 
         PUA  362.786900E-12  -31.414400E-12 
         PU0    7.380991       -4.028575     
       PRDSW   -5              -2.504635     
       PETA0  100.000000E-06  -67.588480E-06 
         PK2   30.964830E-06    3.159047E-03 
         VTM     .025864         .025864     
       PBSWG     .8008           .8          
       MJSWG     .1001           .163262     
       CJSWG  422.000000E-12  330.000000E-12 


Mosfet M_U1_M16, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U1_M16, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U1_M16, model CMOSP: Ps = 0 is less than W
Mosfet M_U1_M1, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U1_M1, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U1_M1, model CMOSN: Ps = 0 is less than W
**** 04/17/10 18:28:15 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "NANDS-test"  [ D:\Documents\TESIS\fiocs\Design\CARACTERIZACION\caracterizacion-pspicefiles\nands\test.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  NEG)    3.3000  (NAND2)    3.3000  (NAND3)    3.3000  (NAND4)    3.3000      

(NAND5)    3.3000  (NAND7)    3.3000  (NAND8)    3.3000  (N00173)    3.3000     

(U1_ND2)    2.7361 (U1_ND3)    2.6961 (U1_ND4)    2.6773 (U1_ND5)    2.6654     

(U1_ND6)    2.6567 (U1_ND7)    2.6499 (U1_ND8)    2.6444 (U2_ND2)    2.7365     

(U4_ND2)    2.7367 (U4_ND3)    2.6967 (U5_ND2)    2.7367 (U5_ND3)    2.6967     

(U5_ND4)    2.6779 (U6_ND2)    2.7366 (U6_ND3)    2.6966 (U6_ND4)    2.6778     

(U6_ND5)    2.6659 (U7_ND2)    2.7363 (U7_ND3)    2.6963 (U7_ND4)    2.6775     

(U7_ND5)    2.6656 (U7_ND6)    2.6569 (U7_ND7)    2.6501 (ENTRADA)    0.0000    




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1         0.000E+00
    V_VDD       -4.530E-09

    TOTAL POWER DISSIPATION   1.49E-08  WATTS



          JOB CONCLUDED

**** 04/17/10 18:28:15 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "NANDS-test"  [ D:\Documents\TESIS\fiocs\Design\CARACTERIZACION\caracterizacion-pspicefiles\nands\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .19
